cmd/verify/network/cmd.go

```diff
@@ -155,10 +156,12 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
 
 	if !cmd.Flags().Changed(subnetIDsFlag) {
 		if cluster != nil {
-			args.subnetIDs = cluster.AWS().SubnetIDs()
-			if len(args.subnetIDs) == 0 {
-				return fmt.Errorf("No subnets on cluster '%s'", cluster.ID())
+			if !helper.IsBYOVPC(cluster) {
+				return fmt.Errorf(
+					"Running the network verifier is only supported for clusters created on existing VPCs")
```

Comment:
```suggestion
					"Running the network verifier is only supported for clusters created on pre existing VPCs prior to cluster creation")
```

Just cause the VPCs do exist for non BYO as well, they just didn't pre existing prior to cluster creation

Reply:
Or just mention only supported for BYO VPC clusters, might be clearer :)